tools/create-package/README.md

```diff
@@ -1,7 +1,11 @@
 # create-package
 
-This library was generated with [Nx](https://nx.dev).
+This library is to create a new package within the `packages` directory.
+
+It sets up the necessary touch points.
+
+Use the `pnpm create-package` script in the root `package.json`
 
 ## Building
 
-Run `nx build create-package` to build the library.
+Run `pnpm nx build create-package` to build the library.
```

Comment:
Sorry if this is a silly question, but what's the difference between `pnpm create-package` and `pnpm nx build create-package`?

Reply:
They are identical. The main difference is that `pnpm create-package` is an alias for `pnpm nx build create-package`. I'm fine referencing the `alias`, however, when looking at the `documentation` for the actual package that is being run, it _may_ be better to reference the actual command.

Open to a different thought though

Reply:
Okay, the reason I brought this up was if they were the same, let's pick one, rather than use both throughout our docs.

Reply:
Lets add this to a new PR for now